hydromt_wflow/wflow.py

```diff
@@ -2593,104 +2601,118 @@ def write_staticmaps(self):
+        self.logger.info(f"Write grid to {fn}")
         mask = ds_out[self._MAPS["basins"]] > 0
         for v in ds_out.data_vars:
             # nodata is required for all but boolean fields
             if ds_out[v].dtype != "bool":
                 ds_out[v] = ds_out[v].where(mask, ds_out[v].raster.nodata)
         ds_out.to_netcdf(fn, encoding=encoding)
-        # self.write_staticmaps_pcr()
+        # self.write_grid_pcr()
+
+    def write_staticmaps(self, **kwargs):
+        """Write staticmaps."""
+        warnings.warn(
+            "write_staticmaps is deprecated. Use 'write_grid' instead. "
+            "Will be removed in hydrom_wflow v0.6.0",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        self.write_grid()
 
     def read_staticmaps_pcr(self, crs=4326, **kwargs):
         """Read and staticmaps at <root/staticmaps> and parse to xarray."""
+        warnings.warn(
+            "read_staticmaps_pcr as a method of this object is deprecated. "
+            "Use 'read_staticmaps_pcr' for the 'pcrm' submodule instead. "
+            "Will be removed in hydrom_wflow v0.5.0",
+            DeprecationWarning,
+            stacklevel=2,
+        )
         if self._read and "chunks" not in kwargs:
             kwargs.update(chunks={"y": -1, "x": -1})
-        fn = join(self.root, "staticmaps", "*.map")
-        fns = glob.glob(fn)
-        if len(fns) == 0:
-            self.logger.warning(f"No staticmaps found at {fn}")
-            return
-        self._staticmaps = open_mfraster(fns, **kwargs)
-        path = join(self.root, "staticmaps", "clim", "LAI*")
-        if len(glob.glob(path)) > 0:
-            da_lai = open_mfraster(
-                path, concat=True, concat_dim="time", logger=self.logger, **kwargs
-            )
-            self.set_staticmaps(da_lai, "LAI")
-
-        # reorganize c_0 etc maps
-        da_c = []
-        list_c = [v for v in self._staticmaps if str(v).startswith("c_")]
-        if len(list_c) > 0:
-            for i, v in enumerate(list_c):
-                da_c.append(self._staticmaps[f"c_{i:d}"])
-            da = xr.concat(
-                da_c, pd.Index(np.arange(len(list_c), dtype=int), name="layer")
-            ).transpose("layer", ...)
-            self.set_staticmaps(da, "c")
-
-        if self.crs is None:
-            if crs is None:
-                crs = 4326  # default to 4326
-            self.set_crs(crs)
+        read_pcr_func(
+            self.root,
+            crs=crs,
+            obj=self,
+            kwargs=kwargs,
+        )
 
     def write_staticmaps_pcr(self):
         """Write staticmaps at <root/staticmaps> in PCRaster maps format."""
+        warnings.warn(
+            "write_staticmaps_pcr as a method of this object is deprecated. "
+            "Use 'write_staticmaps_pcr' for the 'pcrm' submodule instead. "
+            "Will be removed in hydrom_wflow v0.5.0",
+            DeprecationWarning,
+            stacklevel=2,
+        )
```

Comment:
In some places you use `logger.warning` in others `warnings.warn`, I am not sure if there is a functional difference but unless it is deliberate I think it would be best to try and stay consistent. Especially since `logger.warn` seems to be deprecated but `warnings.warn` not. *shakes fist*